src/Modal/AlertModal.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ModalDialog from './ModalDialog';
+
+const AlertModal = ({ children, footerNode, ...props }) => (
+  <ModalDialog
+    {...props}
+    className={classNames('pgn__alert-modal', props.className)}
+  >
+    <ModalDialog.Header>
+      <ModalDialog.Title>{props.title}</ModalDialog.Title>
+    </ModalDialog.Header>
+    <ModalDialog.Body>{children}</ModalDialog.Body>
+    {footerNode && <ModalDialog.Footer>{footerNode}</ModalDialog.Footer>}
+  </ModalDialog>
+);
+
+AlertModal.propTypes = {
+  children: PropTypes.node.isRequired,
+  title: PropTypes.string.isRequired,
+  onClose: PropTypes.func.isRequired,
+  isOpen: PropTypes.bool,
+  hasCloseButton: PropTypes.bool,
+  size: PropTypes.oneOf(['sm', 'md', 'lg', 'xl', 'fullscreen']),
+  variant: PropTypes.oneOf(['default', 'warning', 'danger', 'success', 'dark']),
```

Comment:
Do we not have these listed in a constant somewhere? If no, should we?

Reply:
We could, but our docgen can't parse constants so our props table would just say 'one of VARIANTS' instead. wdyt?